framework/wazuh/agent.py

```diff
@@ -62,12 +62,23 @@ def get_distinct_agents(agent_list=None, offset=0, limit=common.database_limit,
 def get_agents_summary_status(agent_list=None):
     """Counts the number of agents by status.
 
-    :param agent_list: List of agents ID's.
-    :return: WazuhResult.
+    Parameters
+    ----------
+    agent_list : list[str]
+       List of agents ID's
+
+    Returns
+    -------
+    WazuhResult
     """
+    # We don't consider agent 000 in order to get the summary
```

Comment:
It is much more efficient and clean to just send the WazuhDBQueryAgents with q='id!=000', like in https://github.com/wazuh/wazuh/blob/fad97a057deb37dc5c806c0d5e39038de2aa76ad/framework/wazuh/agent.py#L861